apps/alchemist/test/integration/event/event_handler_test.exs

```diff
@@ -0,0 +1,85 @@
+defmodule Alchemist.Event.EventHandlerTest do
+  use ExUnit.Case
+  use Divo
+  use Placebo
+  use Properties, otp_app: :alchemist
+
+  import SmartCity.TestHelper
+  import SmartCity.Event
+  alias SmartCity.TestDataGenerator, as: TDG
+  alias DeadLetter
+
+  @instance_name Alchemist.instance_name()
+  getter(:elsa_brokers, generic: true)
+  getter(:input_topic_prefix, generic: true)
+
+  describe "Ingestion Update" do
+    test "A failing message gets placed on dead letter queue and discarded" do
+      id_for_invalid_ingestion = UUID.uuid4()
+      invalid_ingestion = TDG.create_ingestion(%{id: id_for_invalid_ingestion})
+
+      id_for_valid_ingestion = UUID.uuid4()
+      valid_ingestion = TDG.create_ingestion(%{id: id_for_valid_ingestion})
+      allow(Alchemist.IngestionProcessor.start(invalid_ingestion), exec: fn _ -> raise "nope" end)
+
+      Brook.Event.send(@instance_name, ingestion_update(), __MODULE__, invalid_ingestion)
+      Brook.Event.send(@instance_name, ingestion_update(), __MODULE__, valid_ingestion)
```

Comment:
Publishing a 2nd message to confirm the first isn't stuck in a loop, preventing newer messages from processesing. This is the main use-case of adding these error handlers